packages/quantic/cypress/integration/breadcrumb-manager/breadcrumb-manager.cypress.ts

```diff
@@ -0,0 +1,205 @@
+            Actions.numerciFacet.checkValueAt(index);
+
+            Expect.numerciFacet.displayValues(true);
+            Expect.numerciFacet.numberOfValues(index + 1);
+            Expect.numerciFacet.displayShowMore(false);
+          }
+
+          Actions.numerciFacet.checkValueAt(5);
+
+          Expect.numerciFacet.displayValues(true);
+          Expect.numerciFacet.numberOfValues(5);
+          Expect.numerciFacet.displayShowMore(true);
+
+          Actions.clickShowMoreNumericFacetBreadcrumb();
+
+          Expect.numerciFacet.numberOfValues(6);
+          Expect.numerciFacet.displayShowMore(false);
+        });
+
+        scope('when clearing a value', () => {
+          Actions.clickFirstValueNumericFacetBreadcrumb();
+          cy.wait(InterceptAliases.Search);
+
+          Expect.numerciFacet.numberOfValues(5);
+        });
+      });
+
+      scope('when selecting values in time frame facet', () => {
+        Expect.dateFacet.displayBreadcrumb(false);
+
+        Actions.timeFrameFacet.selectValue('Past 6 months');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.dateFacet.displayBreadcrumb(true);
+        Expect.dateFacet.displayLabel(true);
+        Expect.dateFacet.labelContains('Date');
+        Expect.dateFacet.numberOfValues(1);
+        Expect.dateFacet.breadcrumbValueLabelAtIndexContains('Past 6 months');
+
+        Actions.timeFrameFacet.selectValue('Past year');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.dateFacet.displayBreadcrumb(true);
+        Expect.dateFacet.displayLabel(true);
+        Expect.dateFacet.numberOfValues(1);
+        Expect.dateFacet.breadcrumbValueLabelAtIndexContains('Past year');
+      });
+
+      scope('when clearing all filters', () => {
+        Actions.clickClearFilters();
+
+        Expect.numerciFacet.displayBreadcrumb(false);
+        Expect.dateFacet.displayBreadcrumb(false);
+      });
+
+      scope('when selecting values in category facet', () => {
+        Expect.categoryFacet.displayBreadcrumb(false);
+
+        Actions.categoryFacet.selectChildValue('North America');
+
+        Expect.categoryFacet.displayBreadcrumb(true);
+        Expect.categoryFacet.displayLabel(true);
+        Expect.categoryFacet.labelContains('Country');
+        Expect.categoryFacet.displayValues(true);
+        Expect.categoryFacet.numberOfValues(1);
+        Expect.categoryFacet.breadcrumbValueLabelAtIndexContains(
+          'North America'
+        );
+
+        Actions.categoryFacet.selectChildValue('Canada');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.categoryFacet.numberOfValues(1);
+        Expect.categoryFacet.breadcrumbValueLabelAtIndexContains(
+          'North America / C'
+        );
+
+        Actions.categoryFacet.selectChildValue('British Columbia');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.categoryFacet.numberOfValues(1);
+        Expect.categoryFacet.breadcrumbValueLabelAtIndexContains(
+          'North America / C'
```

Comment:
This string is a bit weird. Almost looks like we're validating a bug.

Reply:
yes

Reply:
We can probably do something better no? Even with ellipsis the innerText is the full string. Cypress won't pick it up?